tendrl/commons/objects/node/flows/stop_services/__init__.py

```diff
@@ -11,6 +11,20 @@ def run(self):
         super(StopServices, self).run()
         services = self.parameters['Services[]']
         for service in services:
+            srv = NS.tendrl.objects.Service(service=service)
```

Comment:
will this be up to date? or do you need to execute the command through ansible and check?

Reply:
I feel it should be fine as it runs service status command to get current status.